Templates/VIPER Module Spec.xctemplate/___VARIABLE_moduleName___ModuleSpec.swift

```diff
@@ -38,7 +38,6 @@ final class ___VARIABLE_moduleName___ModuleSpec: QuickSpec {
                     expect(presenter.view).to(beAKindOf(___VARIABLE_moduleName___ViewController.self))
                     expect(presenter.router).to(beAKindOf(___VARIABLE_moduleName___Router.self))
                     expect(presenter.interactor).to(beAKindOf(___VARIABLE_moduleName___Interactor.self))
-                    expect(presenter.output) === output
```

Comment:
I think we need to keep this

Reply:
yes, we need to keep this line.

Reply:
because we need to check for is the module's output and presenter's output is the same instance or not.

https://github.com/nimblehq/VIPER-Templates/pull/13/files#diff-825b9db831aa8e005dab1bc0f9082cc4R28

Reply:
I have replaced `expect(presenter.output) === output` with `expect(presenter.output) === module.output` which is more explicit in my opinion, What do you guys think?

Reply:
@Dekablade01 @suho  Please review the PR once again.